doc/source/comparison_with_sas.rst

```diff
@@ -0,0 +1,604 @@
+
+   tips[tips['total_bill'] > 10].head()
+
+If/Then Logic:
+~~~~~~~~~~~~~~
+
+In SAS, if/then logic can be used to create new columns.
+
+.. code-block:: none
+
+   data tips;
+       set tips;
+       format bucket $4.;
+
+       if total_bill < 10 then bucket = 'low';
+       else bucket = 'high';
+   run;
+
+The same operation in pandas can be accomplished using
+the ``where`` method from ``numpy``.
+
+.. ipython:: python
+
+   tips['bucket'] = np.where(tips['total_bill'] < 10, 'low', 'high')
+   tips.head()
+
+.. ipython:: python
+   :suppress:
+
+   tips = tips.drop('bucket', axis=1)
+
+Date Functionality
+~~~~~~~~~~~~~~~~~~
+
+SAS provides a variety of functions to do operations on
+date/datetime columns.
+
+.. code-block:: none
+
+   data tips;
+       set tips;
+       format date1 date2 date1_plusmonth mmddyy10.;
+       date1 = mdy(1, 15, 2013);
+       date2 = mdy(2, 15, 2015);
+       date1_year = year(date1);
+       date2_month = month(date2);
+       * shift date to begninning of next interval;
+       date1_next = intnx('MONTH', date1, 1);
+       * count intervals between dates;
+       months_between = intck('MONTH', date1, date2);
+   run;
+
+The equivalent pandas operations are shown below.  In addition to these
+functions pandas supports other Time Series features
+not available in Base SAS (such as resampling and and custom offets) -
+see the :ref:`timeseries documentation<timeseries>` for more details.
+
+.. ipython:: python
+
+   tips['date1'] = pd.Timestamp('2013-01-15')
+   tips['date2'] = pd.Timestamp('2015-02-15')
+   tips['date1_year'] = tips['date1'].dt.year
+   tips['date2_month'] = tips['date2'].dt.month
+   tips['date1_next'] = tips['date1'] + pd.offsets.MonthBegin()
+   tips['months_between'] = (tips['date2'].dt.to_period('M') -
+                             tips['date1'].dt.to_period('M'))
+
+   tips[['date1','date2','date1_year','date2_month',
+         'date1_next','months_between']].head()
+
+.. ipython:: python
+   :suppress:
+
+   tips = tips.drop(['date1','date2','date1_year',
+      'date2_month','date1_next','months_between'], axis=1)
+
+Selection of Columns
+~~~~~~~~~~~~~~~~~~~~
+
+SAS provides keywords in the ``DATA`` step to select,
+drop, and rename columns.
+
+.. code-block:: none
+
+   data tips;
+       set tips;
+       keep sex total_bill tip;
+   run;
+
+   data tips;
+       set tips;
+       drop sex;
+   run;
+
+   data tips;
+       set tips;
+       rename total_bill=total_bill_2;
+   run;
+
+The same operations are expressed in pandas below.
+
+.. ipython:: python
+
+   # keep
+   tips[['sex', 'total_bill', 'tip']].head()
+
+   # drop
+   tips.drop('sex', axis=1).head()
+
+   # rename
+   tips.rename(columns={'total_bill':'total_bill_2'}).head()
+
+
+Sorting by Values
+~~~~~~~~~~~~~~~~~
+
+Sorting in SAS is accomplished via ``PROC SORT``
+
+.. code-block:: none
+
+   proc sort data=tips;
+       by sex total_bill;
+   run;
+
+pandas objects have a ``sort_values`` method, which
```

Comment:
same here as for read_csv (in this case ``` :meth:`~DataFrame.sort_values` ``` is best, the `~` makes you only see the sort_values and not the 'DataFrame')